### PR TITLE
Add tests and structure in symptoms assignation probabilities

### PR DESCRIPTION
**Pull request by satyaog**
# Description

Add tests and structure in symptoms assignation probabilities. The structure will help ensure reproducibility of the datasets and make sure that fields don't get mixed-up in future datasets (next step)

## Type of change

Please delete options that are not relevant.

- [x] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [x] This change requires a documentation update

# How Has This Been Tested?

Please describe the tests that you ran to verify your changes. Provide instructions so we can reproduce. Please also list any relevant details for your test configuration

- [x] python -m run.py test

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes

### CODE REVIEW

File: utils.py
Comment (plstcharles) [inline utils.py]: are these notes related to a variable below, or forgotten in the change?

**Review by plstcharles — APPROVED** [review]
I looked at most changes & tried the tests, everything works on my side. There are quite a few changes to utils.py however, and @teganmaharaj or @mweiss17 or @pg2455 should review these to make sure they don't break other stuff before merging

File: utils.py
Comment (satyaog) [reply utils.py]: Added docstrings to make this clear

**Pull request closed without merging**
